Modules/Relationships/src/Relationships.Application/Relationships/Commands/AcceptRelationshipChangeRequest/AcceptRelationshipChangeRequestCommand.cs

```diff
@@ -5,7 +5,7 @@ namespace Backbone.Modules.Relationships.Application.Relationships.Commands.Acce
 
 public class AcceptRelationshipChangeRequestCommand : IRequest<AcceptRelationshipChangeRequestResponse>
 {
-    public RelationshipId Id { get; set; }
-    public RelationshipChangeId ChangeId { get; set; }
-    public byte[] ResponseContent { get; set; }
+    public required RelationshipId Id { get; set; }
+    public required RelationshipChangeId ChangeId { get; set; }
+    public required byte[] ResponseContent { get; set; }
```

Comment:
`Response` is defined as `NULL` in the database - I thing this should not be `required` but optional (nullable). Moreover, in `Modules.Relationships.Application.Relationships.Commands.AcceptRelationshipChangeRequest.AcceptRelationshipChangeRequestCommandValidator` you have the following rule:
```csharp
        RuleFor(c => c.ResponseContent).NumberOfBytes(0, 10.Mebibytes());
```

Furthermore, in `Modules.Relationships.Application.Relationships.Commands.AcceptRelationshipChangeRequest.Handler` you have this call (`changeRequest` is of type `AcceptRelationshipChangeRequestCommand`):
```csharp
        var change = relationship.AcceptChange(changeRequest.ChangeId, _userContext.GetAddress(), _userContext.GetDeviceId(), changeRequest.ResponseContent);
```

The `AcceptChange` method has the following signature (note that `content` is nullable):
```csharp
    public RelationshipChange AcceptChange(RelationshipChangeId changeId, IdentityAddress acceptedBy, DeviceId acceptedByDevice, byte[]? content)
```